v0.6.0 Fails to Build due to atrium_api Version Clash

Hello, hope you're well. I tried compiling `examples/sqlite`, and got the following error. It appeared that there was a conflicting dependency on `atrium_api` between `0.24.10` and `0.25.2`. A quick look in `cargo tree` showed that `skyfeed` depended on `0.24.10` but `jetstream-oxide` depended on `0.25.2`. 

```
   Compiling skyfeed v0.6.0 (/home/ben/git/skyfeed)                                                                                                                                           
error[E0308]: mismatched types                                                                                                                                                                
   --> /home/ben/git/skyfeed/src/feed.rs:149:49                                                                                                                                               
    |                                                                                                                                                                                         
136 |                         match commit {                                                                                                                                                  
    |                               ------ this expression has type `CommitEvent`                                                                                                             
...                                                                                                                                                                                           
149 |                                         record: KnownRecord::AppBskyFeedPost(record),                                                                                                   
    |                                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `atrium_api::record::KnownRecord`, found `KnownRecord`                                    
    |                                                                                                                                                                                         
    = note: `KnownRecord` and `atrium_api::record::KnownRecord` have similar names, but are actually distinct types                                                                           
note: `KnownRecord` is defined in crate `atrium_api`                                                                                                                                          
   --> /home/ben/.cargo/registry/src/index.crates.io-6f17d22bba15001f/atrium-api-0.24.10/src/record.rs:5:1                                                                                    
    |                                                                                                                                                                                         
5   | pub enum KnownRecord {                                                                                                                                                                  
    | ^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                    
note: `atrium_api::record::KnownRecord` is defined in crate `atrium_api`                                                                                                                      
   --> /home/ben/.cargo/registry/src/index.crates.io-6f17d22bba15001f/atrium-api-0.25.2/src/record.rs:5:1
    |
5   | pub enum KnownRecord {
    | ^^^^^^^^^^^^^^^^^^^^
    = note: perhaps two different versions of crate `atrium_api` are being used?
```

PR follows.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -37,7 +37,7 @@ path = "examples/sqlite/src/main.rs"
 required-features = ["examples"]
 
 [dependencies]
-atrium-api = "=0.24.10"
+atrium-api = "0.25.2"
 dotenv = "0.15.0"
 env_logger = "0.11.5"
 jetstream-oxide = "0.1.1"

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -37,10 +37,10 @@ path = "examples/sqlite/src/main.rs"
 required-features = ["examples"]
 
 [dependencies]
-atrium-api = "0.25.2"
+atrium-api = "=0.25.2"
 dotenv = "0.15.0"
 env_logger = "0.11.5"
-jetstream-oxide = "0.1.1"
+jetstream-oxide = "=0.1.2"
 log = "0.4.22"
 serde = "1.0.215"
 serde_json = "1.0.133"

--- a/src/bin/my_did.rs
+++ b/src/bin/my_did.rs
@@ -1,6 +1,6 @@
 use anyhow::Result;
 use atrium_api::{
-    agent::{store::MemorySessionStore, AtpAgent},
+    agent::atp_agent::{store::MemorySessionStore, AtpAgent},
     types::string::Handle,
 };
 use clap::Parser;

--- a/src/bin/publish.rs
+++ b/src/bin/publish.rs
@@ -1,6 +1,7 @@
 use atrium_api::com::atproto::repo::put_record::InputData;
+use atrium_api::types::string::RecordKey;
 use atrium_api::{
-    agent::{store::MemorySessionStore, AtpAgent},
+    agent::atp_agent::{store::MemorySessionStore, AtpAgent},
     app::bsky::feed::generator::RecordData,
     types::{
         string::{Datetime, Did, Handle, Nsid},
@@ -50,6 +51,7 @@ async fn main() {
     let handle = args.handle;
     let password = args.app_password;
     let feed_host_name = args.hostname;
+    let record_key = RecordKey::new(args.name.to_owned()).expect("Invalid record key name.");
 
     println!("Logging in...");
 
@@ -122,7 +124,7 @@ async fn main() {
                 repo: atrium_api::types::string::AtIdentifier::Did(
                     publisher_did.to_owned().did.clone(),
                 ),
-                rkey: args.name.to_owned(),
+                rkey: record_key,
                 swap_commit: None,
                 swap_record: None,
                 validate: None,

--- a/src/bin/unpublish.rs
+++ b/src/bin/unpublish.rs
@@ -1,7 +1,7 @@
 use anyhow::Result;
 use atrium_api::{
-    agent::{store::MemorySessionStore, AtpAgent},
-    types::string::{Handle, Nsid},
+    agent::atp_agent::{store::MemorySessionStore, AtpAgent},
+    types::string::{Handle, Nsid, RecordKey},
 };
 use clap::Parser;
 
@@ -28,6 +28,8 @@ async fn main() -> Result<()> {
 
     let handle = args.handle;
     let password = args.app_password;
+
+    let record_key = RecordKey::new(args.name.to_owned()).expect("Invalid record key name.");
 
     println!("Logging in");
 
@@ -62,7 +64,7 @@ async fn main() -> Result<()> {
                 repo: atrium_api::types::string::AtIdentifier::Did(
                     publisher_did.to_owned().did.clone(),
                 ),
-                rkey: args.name.to_owned(),
+                rkey: record_key,
                 swap_commit: None,
                 swap_record: None,
             }

--- a/src/feed.rs
+++ b/src/feed.rs
@@ -283,7 +283,7 @@ async fn get_feed_skeleton<Handler: FeedHandler>(
         .serve_feed(Request {
             cursor: query.cursor.clone(),
             feed: query.feed.clone(),
-            limit: query.limit,
+            limit: query.limit.map(|l| u8::from(l)),
         })
         .await;
     Ok::<warp::reply::Json, warp::Rejection>(warp::reply::json(&FeedSkeleton {

--- a/src/feed.rs
+++ b/src/feed.rs
@@ -283,7 +283,7 @@ async fn get_feed_skeleton<Handler: FeedHandler>(
         .serve_feed(Request {
             cursor: query.cursor.clone(),
             feed: query.feed.clone(),
-            limit: query.limit.map(|l| u8::from(l)),
+            limit: query.limit.map(u8::from),
         })
         .await;
     Ok::<warp::reply::Json, warp::Rejection>(warp::reply::json(&FeedSkeleton {

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -7,4 +7,7 @@ mod utility_models;
 
 pub use feed::Feed;
 pub use feed_handler::FeedHandler;
-pub use models::*;
+pub use models::{
+    Cid, Did, Embed, ExternalEmbed, FeedResult, ImageEmbed, Label, MediaEmbed, Post, QuoteEmbed,
+    Request, Uri, VideoEmbed,
+};

--- a/src/models.rs
+++ b/src/models.rs
@@ -3,7 +3,7 @@ use atrium_api::{
         embed::record_with_media::MainMediaRefs,
         feed::post::{RecordEmbedRefs, RecordLabelsRefs},
     },
-    types::{BlobRef, LimitedNonZeroU8, Object, TypedBlobRef, Union},
+    types::{BlobRef, Object, TypedBlobRef, Union},
 };
 use chrono::{DateTime, Utc};
 use log::trace;
@@ -12,7 +12,7 @@ use log::trace;
 pub struct Request {
     pub cursor: Option<String>,
     pub feed: String,
-    pub limit: Option<LimitedNonZeroU8<100>>,
+    pub limit: Option<u8>,
 }
 
 #[derive(Debug, Clone)]

